spec/features/people_skill_search_spec.rb

```diff
@@ -2,34 +2,117 @@
 
 describe :people_skills do
   describe 'People Search', type: :feature, js: true do
-    let(:skill) { skills(:rails).attributes }
+    let(:rails) { skills(:rails).attributes }
```

Comment:
```suggestion
    let(:skill) { skills(:rails).attributes }
```

I find it rather confusing if you just read rails further down in the spec ... `rails_skill` would also also be a suitable name maybe